pkg/doc/verifiable/presentation_test.go

```diff
@@ -46,31 +49,7 @@ const validPresentation = `
       }
     }
   ],
-  "holder": "did:example:ebfeb1f712ebc6f1c276e12ec21",
-  "proof": {
-    "type": "Ed25519Signature2018",
-    "created": "2020-01-21T16:44:53+02:00",
-    "proofValue": "eyJhbGciOiJSUzI1NiIsImI2NCI6ZmFsc2UsImNyaXQiOlsiYjY0Il19..kTCYt5XsITJX1CxPCT8yAV-TVIw5WEuts01mq-pQy7UJiN5mgREEMGlv50aqzpqh4Qq_PbChOMqsLfRoPsnsgxD-WUcX16dUOqV0G_zS245-kronKb78cPktb3rk-BuQy72IFLN25DYuNzVBAh4vGHSrQyHUGlcTwLtjPAnKb78"
-  }
-}
-`
-
-// nolint:lll
-const validEmptyPresentation = `
```

Comment:
not needed anymore?

Reply:
I moved `WithPresRequireVC()` and new `WithPresRequireProof()` options to the `credential_testsuite.go` (under `testsuite` tag). But yeah, I did not move the tests. Corrected now - I also added the test for the empty `proof` field. See `TestNewPresentationWithEmptyFields()`.